### ISSUE TITLE
Can't open the .exe file, says: "You may not have the appropriate permission to access the item"

### ISSUE THREAD

**Issue by Birizinho** [SteamGridDB/steam-rom-manager#38]
Just for the record, I'm [that guy on Reddit](https://www.reddit.com/r/emulation/comments/6sihru/steam_rom_manager_210_is_now_released/dlgkfld/?context=3) that asked for a solution on this problem.

I tried multiple tutorials to give the only user present in my PC the "maximum administrator permission" (If it didn't already have it, I didn't get much of the point there) and tried to open multiple releases of SRM, but still I didn't manage to open the .exe file without the error message.

However, when I uninstalled the app through IObit Uninstaller, I noticed that it says: "Steam ROM Manager 2.1.1 (only current user)". Is the "only current user" just purely aesthetic or it has anything to do with how the program handles the permissions by user? Is that possible to revert that so all user (or at least the administrator user) are able to universally modify it?

EDIT: Something that I also found is that this is the only program that places a .exe file on C:\Users\\[USERNAME]\AppData\Local (which is more reserved to logs, text files or complimentary files for their respective programs rather than the .exe file itself) while most of the programs that I have are located on C:\Program Files (x86) or C:\Users\\[USERNAME]. I'm not sure if that point is relevant on my problem since I tried to extrat the .zip files of the older releases of SRM on those directories and I still received the error message; but I thought that it would be something interesting to point out.

**Comment by FrogTheFrog** [comment]
@Birizinho This is the famous WTF error from old times. Unfortunately, I can't do anything on my end as it is Windows error and basically no one knows what causes it. You can try extracting executable from the installer with 7-zip or similar programs. However, I doubt it will work since the old version did not work.

Finally, you could try [this](https://www.reddit.com/r/Windows10/comments/45to0g/windows_cannot_access_the_specified_device_path/) solution.

Good luck!

**Comment by HEspoke** [comment]
@Birizinho Another thing that I just realized is that you do not have to use the EXE to run SRM. This solution will be the LONG way around for a new user but it should fix your issue with Windows. You could compile/build the program using the [Developer](https://github.com/FrogTheFrog/steam-rom-manager#for-developers) way of running the program. It will take some leg work on your end by downloading the programs needed to compile the program listed in the  [Developer](https://github.com/FrogTheFrog/steam-rom-manager#for-developers) section. But if Windows is locking your out of permissions to run the EXE build then I dont know what other option you have. 
Basically, If I can figure out how to compile this program and run it then ANYONE can do it. You just gotta put some work into learning how to do it. =)
